CHANGELOG.md

```diff
@@ -42,6 +43,8 @@ tech changes will usually be stripped from release notes for the public
         -   This should now more properly snap shapes that are larger than 1x1
 -   Ruler Tool:
     -   Snap now properly works for hex grids
+    -   Grid mode now starts count from 0 instead of -1
+    -   Grid mode correctly counts total cells when using multiple rulers
```

Comment:
As grid mode is new, any fixes to it should not be included in the changelog as they never existed compared to the previous version.